controlpanel/settings/common.py

```diff
@@ -499,3 +503,26 @@
 # -- Airflow
 AIRFLOW_SECRET_KEY = os.environ.get('AIRFLOW_SECRET_KEY')
 AIRFLOW_FERNET_KEY = os.environ.get('AIRFLOW_FERNET_KEY')
+
+REDIS_HOST = os.environ.get("REDIS_HOST")
+REDIS_PORT = os.environ.get("REDIS_PORT")
+REDIS_PASSWORD = os.environ.get("REDIS_PASSWORD")
```

Comment:
These settings are overriding the existing `REDIS_` settings earlier in this file (lines 439 - 441)

Reply:
Removed